native/cocos/platform/mac/modules/SystemWindow.mm

```diff
@@ -24,41 +23,74 @@ of this software and associated engine source code (the "Software"), a limited,
 ****************************************************************************/
 
 #include "platform/mac/modules/SystemWindow.h"
-#import <AppKit/AppKit.h>
-#include "platform/BasePlatform.h"
-#include "platform/interfaces/modules/IScreen.h"
-
-#if CC_EDITOR
+#include "platform/mac/ViewController.h"
+#include "base/Log.h"
+#include "base/Macros.h"
+
+// SDL headers
+#include <functional>
+#include "SDL2/SDL.h"
+#include "SDL2/SDL_main.h"
+#include "SDL2/SDL_syswm.h"
+#include "engine/EngineEvents.h"
+#include "platform/SDLHelper.h"
```

Comment:
Add header files as the sequences:
- system related header files
- 3rd libs
- engine related